src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/LocalsTests.cs

```diff
@@ -3766,6 +3766,65 @@ static void Main()
             });
         }
 
+        [Fact]
+        public void AssignDefaultToLocal()
```

Comment:
Consider moving to `ExpressionCompilerTests`. `LocalsTests` was, at least initially, intended for testing `CompileGetLocals`.